src/style/stars.css

```diff
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+  
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
```

Comment:
last star shouldn't have margin. Use `:not(:last-child)`

Reply:
isn't fixed. 
It should looks like 
```
.stars__star:not(:last-child) {
  margin-right: 4px;
}
```

Reply:
not fixed 
https://github.com/mate-academy/layout_product-cards/pull/3048/files#r1200840477